build.fsx

```diff
@@ -32,6 +32,38 @@ Target.create "Clean" (fun _ ->
     Git.CommandHelper.directRunGitCommandAndFail "./Content" "clean -fxd"
 )
 
+Target.create "BuildWebPackConfig" (fun _ ->
+    let srcDir = "paket-files/fable-compiler/webpack-config-template/webpack.config.js"
+    let destDir = "Content/src/Client/webpack.config.js"
+    Shell.copyFile destDir srcDir
+
+    let devServerProxy =
```

Comment:
Is this for supporting project which include a server ?

If we want to lower the complexity, I would expect to use the template feature to choose between 2 `webpack.config.js` file when installing the template.

More or less like it's done for `paket.lock` file